src/native/corehost/hostmisc/utils.h

```diff
@@ -120,7 +120,7 @@ template<typename T>
 size_t to_size_t_dbgchecked(T value)
 {
     assert(value >= 0);
-    assert(value < static_cast<T>(std::numeric_limits<size_t>::max()));
+    assert(value < std::numeric_limits<size_t>::max());
```

Comment:
Accidental commit?

Reply:
No, it actually fails without this change. `static_cast<int>(size_t::max())` is `-1`, so this assert always fails in debug.

Reply:
Never mind, fixed in main